### PR TITLE
intial files have been committed

### PR DESCRIPTION
**Pull request by CodeProcastinator**

### CODE REVIEW

**Review by CodeProcastinator — COMMENTED** [review]
C.A.L.M project is added

File: README.md [hunk 1]
```diff
@@ -1,34 +1,123 @@
...
+- **Python 3.12** or higher
+- **pip** (Python package installer)
+
+---
+
+### Installation
+
+1. **Clone the Repository**:
+   ```bash
+   git clone https://github.com/CodeProcastinator/Daytona-Streamlit-Gemini-Project/tree/main
```
Comment (idagelic) [inline README.md]: I believe these first two steps should be replaced with just a `daytona create` command call and by adding the requirements install through the devcontainer hooks. Either way, the repo URL should be the new repo under daytonaio org

**Review by idagelic — CHANGES_REQUESTED** [review]
Please tend to the requested change and also sign off the commit
```
git commit --amend --signoff
git push -f
```

File: README.md (hunk 1 above)
Comment (CodeProcastinator) [reply README.md]: can recheck now

**Review by idagelic — CHANGES_REQUESTED** [review]
image

This doesn't seem to look right. Also, please use the daytonio org repo (the one you're opening a PR to) instead of your personal repo

**Review by idagelic — CHANGES_REQUESTED** [review]
Please create a .devcontainer folder and move the devcontainer.json file there

**Review by idagelic — APPROVED** [review]

**Pull request merged**